packages/apps/manufacturing/src/components/organisms/ConsumedProductGlobalCard/ConsumedProductGlobalCard.tsx

```diff
@@ -84,9 +86,28 @@ const ConsumedProductGlobalCard = ({
   }, [handleTimeOut, incrementVisible, addedQty]);
 
   return (
-    <View style={styles.globalContainer}>
+    <ActionCard
+      translator={I18n.t}
+      actionList={[
+        {
+          iconName: 'geo-alt-fill',
+          onPress: onLocationPress,
+          large: !isSubOF,
+        },
+        {
+          iconName: 'diagram-3-fill',
+          onPress: onSubOfPress,
+          hidden: !isSubOF,
+        },
+        {
+          iconName: 'plus-lg',
+          onPress: handleIncrement,
+          hidden: disableMore,
+          iconColor: Colors.primaryColor.background,
+          large: true,
+        },
```

Comment:
Please add helper on actions : 
* "See availability" / "Voir la disponibilité"
* "See sub MO" / "Voir les sous OF"
* "Increment quantity" / "Augmenter la quantité"